examples/fastify-server/package.json

```diff
@@ -10,7 +10,9 @@
     "dev": "run-p dev:* --print-label",
     "start:server": "node dist/server/index.js",
     "start:client": "yarn wait-server && node dist/client/index.js",
-    "start": "run-p start:* --print-label"
+    "start": "run-p start:* --print-label",
+    "test-dev": "start-server-and-test 'ts-node src/server' 2022 'ts-node src/client'",
+    "test-start": "start-server-and-test 'node dist/server' 2022 'node dist/client'"
```

Comment:
Unsure why this doesn't wanna run - @skarab42, ideas?

Reply:
no ideas I'll look tomorrow

Reply:
Thanks! Feel free to update the draft and convert to a PR if you find a solution

Reply:
Fastify does not respond to HEAD requests from start-server-and-test without setting explicitly a route for it. So we need to use `http-get://` prefix in front of port number.